akalisten/templates/lotsude/index.j2

```diff
@@ -90,9 +111,11 @@
     </div>
      <footer>
          <div>
-            <p>Mit ❤️‍🔥 gebaut von 🦌 und Henry</p>
+            <p class="icon-text"><i data-lucide="database-backup" class="lucide-icon"></i>Zuletzt aktualisiert: {{ now.strftime("%d.%m.%Y %H:%M:%S") }}</p>
+            <p class="icon-text">Mit<i data-lucide="heart-handshake" class="lucide-icon"></i>gebaut von Hirsch und Henry</p>
```

Comment:
`display: flex` does not seem to be the solution for texts with inline-icon.

![image](https://github.com/user-attachments/assets/e1c09098-8105-46bf-828a-39c77edb11eb)

It would look better this way:
![image](https://github.com/user-attachments/assets/1655c5dd-b252-445d-af9e-660996f0d5c3)

I just haven't thought of a sensible way to implement it yet.